news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md

```diff
@@ -27,7 +27,7 @@ This edition of the Core Weekly report highlights changes in PrestaShop's core c
 
 ### Core
 
-* [#15755](https://github.com/PrestaShop/PrestaShop/pull/15755): CLDR Specification\Number should be immutable, by [@Progi1984](https://github.com/Progi1984)
+* [#15755](https://github.com/PrestaShop/PrestaShop/pull/15755): Make CLDR Specification\Number immutable, by [@Progi1984](https://github.com/Progi1984)
```

Comment:
we should not change the title of the pull request only in the core weekly. if we do that, we should also edit the title on GitHub, too, so both match.